tests/main.cpp

```diff
@@ -7,8 +7,32 @@ Date:   10/18/2019
 #include "PersonTests.h"
 #include "DialogueLineTests.h"
 #include "DialogueTests.h"
+#include "../source/DialoguePlayer.h"
 #include <vector>
 
+void testDialoguePlayer()
+{
+    Person kef("1204432", "Kefin", "Sajan", Date::now(), true);
+    Person bloom("1204433", "Jonah", "Bloom", Date::now(), true);
+    Person msMorfe("1204434", "Gabriela", "Morfe", Date::now(), false);
+    Person zofranov("1204435", "Markov", "Zofranov", Date::now(), true);
+    Dialogue dialogue( {
+        DialogueLine( "Hello, everyone. Thanks for attending.", kef ),
```

Comment:
Remove additional spaces after and before the parentheses. Do so until end of function.